backend/main.go

```diff
@@ -37,8 +22,15 @@ func main() {
 		),
 	)
 
-	beego.AddNamespace(writeNs)
-	beego.AddNamespace(readNs)
+	release := beego.NewNamespace("/api",
+		beego.NSNamespace("/v1/release",
+			beego.NSBefore(controllers.Auth),
+            beego.NSRouter("/", &controllers.WorkloadController{}, "post:ReleaseWorkloads"),
+		),
+	)
+
+	beego.AddNamespace(apiNs)
+	beego.AddNamespace(release)
```

Comment:
Worked like a charm, *BUT* I had to change the namespace order (no changes to frontend needed)...
Let's keep it that way, for now, next new "read" feature we should implement a split route